pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -152,25 +156,29 @@ func (s *srv) handleDiscover(c echo.Context) error {
 	return nil
 }
 
+var euiHexPattern = regexp.MustCompile("[a-f0-9A-F]{16}")
+
 func (s *srv) handleTraffic(c echo.Context) error {
 	var sessionID int32
 	id := c.Param("id")
 	auth := c.Request().Header.Get(echo.HeaderAuthorization)
 	ctx := c.Request().Context()
+	md := metadata.MD{}
```

Comment:
If you apply https://github.com/TheThingsNetwork/lorawan-stack/pull/1077#discussion_r309272681 then it makes sense to do the same here.